docs/Usage/9_Troubleshooting.rst

```diff
@@ -0,0 +1,29 @@
+Troubleshooting
```

Comment:
please add a link to issues page of the github repo as:
a) a place to look for solutions as well
b) post new issues

Reply:
Is the current addendum satisfactory? Also Roboy_Parser has its issues page disabled, is that on purpose?